packages/path_provider/path_provider_foundation/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,7 +8,7 @@
 
 /* Begin PBXBuildFile section */
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
-		3380327729784D96002D32AE /* RunnerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3380327629784D96002D32AE /* RunnerTests.swift */; };
+		33258D7929818305006BAA98 /* RunnerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 33258D7729818302006BAA98 /* RunnerTests.swift */; };
```

Comment:
My previous change still had a folder reference to RunnerTests/, but that folder doesn't exist any more so the Xcode UI was really confused. I re-created RunnerTests as a non-folder group, which is why there are changes in the project.